apps/bfd-pipeline/bfd-pipeline-ccw-rif/src/main/java/gov/cms/bfd/pipeline/ccw/rif/extract/s3/DataSetManifest.java

```diff
@@ -342,21 +358,29 @@ private DataSetManifestEntry() {
       this.exportType = null;
     }
 
-    /** @return the {@link DataSetManifest} that this {@link DataSetManifestEntry} is a part of */
+    /**
+     * Gets the {@link #parentManifest}.
+     *
+     * @return the {@link DataSetManifest} that this {@link DataSetManifestEntry} is a part of
+     */
     public DataSetManifest getParentManifest() {
       return parentManifest;
     }
 
     /**
+     * Gets the {@link #name}.
+     *
      * @return the name of the S3 object/file that is represented by this {@link
      *     DataSetManifestEntry}, which is effectively a relative S3 key (relative to this <code>
-     *     manifest.xml</code> object's key, that is)
+     *      manifest.xml</code> object's key, that is)
```

Comment:
Probably an auto-formatted issue, right?
```suggestion
     *     manifest.xml</code> object's key, that is)
```

Reply:
Yeah the build formatter seems to want to add space; it doesnt seem to like the brackets. Not worth fixing further, but the modern style is to use the inline brackets instead of the html-style tags anyhow for the future.